### PR TITLE
CONCF-568 | Fix local navigation for breakpoints for tablet landscape

### PR DESCRIPTION
**Pull request by vforge**
/cc @hakubo @Warkot

### CODE REVIEW

File: skins/oasis/css/core/breakpoints-layout.scss [hunk 1]
```diff
@@ -262,11 +262,13 @@
...
 
 				.subnav-2 {
+					left: 10px;
 					width: auto;
 				}
 
 				.navbackground {
-					 width: 768px;
+					left: 10px;
+					width: 768px;
```
Comment (Warkot) [inline skins/oasis/css/core/breakpoints-layout.scss]: You could use `$content-width-small` variable.

File: skins/oasis/css/core/breakpoints-layout.scss [hunk 2]
```diff
@@ -262,11 +262,13 @@
 				top: 20px;
 
 				.subnav-2 {
+					left: 10px;
 					width: auto;
 				}
 
 				.navbackground {
-					 width: 768px;
+					left: 10px;
```
Comment (Warkot) [inline skins/oasis/css/core/breakpoints-layout.scss]: Isn't this `$padding-width-small-plus`?

Comment (vforge) [inline skins/oasis/css/core/breakpoints-layout.scss]: No

Comment (Warkot) [inline skins/oasis/css/core/breakpoints-layout.scss]: k

Comment (vforge) [inline skins/oasis/css/core/breakpoints-layout.scss]: Let me explain - `$padding-width-small-plus` is connected to difference between page's width and content's width - this is not.

Comment (Warkot) [inline skins/oasis/css/core/breakpoints-layout.scss]: thx

**Comment by hakubo** [comment]
LGTM

**Comment by Warkot** [comment]
LGTM :water_buffalo:

**Pull request merged**